package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "cunningham",
+  "version": "0.0.0",
+  "private": true,
+  "workspaces": [
+    "apps/*",
+    "packages/*"
+  ],
+  "scripts": {
+    "build": "turbo run build",
+    "dev": "turbo run dev --parallel",
+    "test": "turbo run test",
+    "lint": "turbo run lint",
+    "deploy": "turbo run deploy",
+    "format": "prettier --write \"**/*.{ts,tsx,md}\""
```

Comment:
I'll add js and jsx files into the scope of prettier.
To prevent to format dist outputs, I'll also add a `.prettierignore` at the root of the project with the following content : 
```
dist/
```